### DISCUSSION TITLE
Writing out inferred model

### DISCUSSION THREAD

**Discussion by AbhaMoitra** [SemanticApplicationDesignLanguage/sadl#713] [Q&A]
Can I write out the inferred model after the rules have been executed. The inferred model would include what it started with as well as additional inferred info. The output format could be OWL or some other format.

**Comment by crapo** [comment]
The inferred model can be written to a file using the "Print: Model." statement. See the Topical Index (http://semanticapplicationdesignlanguage.github.io/sadl/TopicalIndex.html), Print Statement. Of course one could also do a SPARQL query to retrieve all of the triples in the inferred model and then copy from the console window or use the Write statement to send them to a file.

**Reply by AbhaMoitra** [reply]
Thank you very much!
